.github/workflows/darts-cifar10-e2e-test.yaml

```diff
@@ -2,6 +2,9 @@ name: E2E Test with darts-cnn-cifar10
 on:
   - pull_request
 
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need GITHUB_TOKEN? I don't think, it is exposed to pull_request type for forked repos anways.